backend/controllers/UserController.ts

```diff
@@ -0,0 +1,37 @@
+import { Response, Request } from "express";
+import User from "../models/UserModel";
+
+const UserController = {
+  async getUserData(req: Request, res: Response) {
+    try {
+      const userToken = req.params.userToken;
+
+      const userData = (await User.find(
+          {
+            userToken: userToken
```

Comment:
An email is stored in a token. Just get an email from token using jsonwebtoken package and secret key (stored in process. env) We shouldn't store tokens inside our DB, as it expire in one day.